### ISSUE TITLE
About Input Lag & 144hz Monitors

### ISSUE THREAD

**Issue by MrKarpuz** [haxball/haxball-issues#999]
Since Basro does nothing to make this game fair, I'm thinking to buy a high refresh rate monitor to get rid of input lag. I am %100 sure there is a huge difference between 60-144, using 60 on a laptop now. I need to know if there's a difference between 144 and 240 hz so I can buy whichever is the best. If 144 is enough please let me know because my pc wont have over 150 fps in pubg or cs go so 240 will be all haxball investment. thanks

**Comment by kingjasko** [comment]
240hz is obviously gonna be better for esports games, but you have to also consider panel type and response time. You can get an affordable 144hz IPS display from brands like acer and aoc for sub 200€ with pretty good response times. I think a jump from 144hz to 240hz isnt as big as from 60hz to 144hz. So yeah, if you plan on just using this 1 monitor then I'd recommend you to get the 144hz ips display with like 3-5ms response time unless if you are a hardcore professional esports player haha, then every bit of advantage will matter.

**Comment by hax-ops** [comment]
75hz is enough to play. It's all on you. Yeah there is difference but if you don't know how to play, even 10000hz quantum monitor can't help you.

**Comment by MrKarpuz** [comment]
@hax-ops I never saw a noob complaining about input lag. I am doing my best now but input lag puts some limits, wanna feel better, faster. @kingjasko thanks for the reply I can afford 144hz 1ms or 240hz 1ms, if there is a difference of course i will pick 240hz. @hax-ops stop listening people who talks like you, every 0.1ms is imporant on online competitive games, we all know how to play this game.

**Comment by hax-ops** [comment]
> @hax-ops I never saw a noob complaining about input lag. I am doing my best now but input lag puts some limits, wanna feel better, faster. @kingjasko thanks for the reply I can afford 144hz 1ms or 240hz 1ms, if there is a difference of course i will pick 240hz. @hax-ops stop listening people who talks like you, every 0.1ms is imporant on online competitive games, we all know how to play this game.

well, get yourself a 240hz and lets play 1on1 we'll see who is winning. xd

bruh you think im saying this thing so random but you don't know I play with players who got 144 and 240hz monitors everyday and im winning against them with 60hz.

so hz is not important **at all** if you are below my or anyone's level me/they still able to destroy you with 60hz against your so called 240hz 1ms

**Comment by kingjasko** [comment]
I suggest you watch this, it might change your mind @hax-ops https://www.youtube.com/watch?v=OX31kZbAXsA

**Comment by hax-ops** [comment]
> I suggest you watch this, it might change your mind @hax-ops https://www.youtube.com/watch?v=OX31kZbAXsA

i agree its giving you an advantage, but people like mrkarpuz thinks when they get brand new 240 they will start to play like misaj or any pro player. well its not true, krol also said that in past.

let s1mple play with 60hz he would still destroy everyone in the game. so good players adapt to everything lol maybe you dont believe me but i have friend who plays since 2013 with 30 fps mode dude banging everyone in the game and he is like legend, thats what im talking about. good players can even play with 30 fps. and still destroy 240 fps players. im telling you dont think wrong. dont spent your money on this game, get better on 60hz, play more, train yourself more.

**Comment by MrKarpuz** [comment]
@hax-ops dude youre just talking to tell us you're a pro, ok i accept youre better then me and jasko but I want to make you understand one simple thing, a real fact in haxball, this game is not like cs go, not optimized, in cs go 60hz vs 240hz can be rival, because game never implies input lag the only difference is visual, but in haxball 60hz vs 240 hz is different more than just visual, game gets your keystrokes like 1 hour later in 60hz, even win10 vs linux is different in this game there is no fair, I know misaj is a good player and he will be better then me even if I get 500hz monitor because he is more experienced, think about two misaj against each oher one is playing 60 the other 240, 60 misaj has no chance against the other believe me.

**Comment by Mirac35** [comment]
144 al geç kardeşim bu andavallarla da vakit kaybetme.
